_layouts/doc.html

```diff
@@ -1,6 +1,11 @@
 <!DOCTYPE html>
 <html lang="{{ site.lang }}">
 <head>
+{%- if page.external-link -%}
+  <meta http-equiv="refresh" content="0; url={{ page.external-link }}">
+</head>
+</html>
+{%- else -%}
```

Comment:
Couldn't this be achieved a bit more locally by setting the layout of the docs sub-page to "redirect"? Having this in the global "doc" layout is a bit unexpected from a software architecture point of view. Things like this make it harder to maintain in the future.

Reply:
Right. I guess we could, but with the current set-up I believe the user would see a flash of the redirect page, which isn't great for internal purposes. So if we take that approach, I would remove the delay (if it's there).

Reply:
There is no delay. The redirect page uses the exact same meta tag that you added here now.

Reply:
Then I don't know why I remember seeing flashes of the Redirect page. Will update this accordingly.